pkgs/development/python-modules/loguru-logging-intercept/default.nix

```diff
@@ -0,0 +1,30 @@
+{
+  lib,
+  buildPythonPackage,
+  setuptools,
+  loguru,
+  fetchFromGitHub,
+}:
+buildPythonPackage {
+  pname = "loguru-logging-intercept";
+  version = "0.1.4-unstable";
+
+  src = fetchFromGitHub {
+    owner = "MatthewScholefield";
+    repo = "loguru-logging-intercept";
+    rev = "9252ac889d4af08fd11d760d92d56a9de93a004a";
+    hash = "sha256-lHACnXhXblNfzBYZ/3BTVupp6b9257jDN5kY3xvfVUg=";
+  };
```

Comment:
There's [sdist on Pypi](https://pypi.org/project/loguru-logging-intercept/#files), and no difference in loguru_logging_intercept.py.

We could fetch from Pypi to follow the version.